config.py

```diff
@@ -13,8 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Ivy test files
+IVY_FILE_LIST = ["ivy/ivy.xml",
```

Comment:
I think that Fireant should look for all `ivy.xml` in the project rather than the user specifying them ina file like this.

Reply:
I'll remove this since it won't be used (this was mainly just for testing). The finished code will iterate through the `src/plugin` folder for ivy.xml files along with the main one (`ivy/ivy.xml`). The code for that part is here https://github.com/fireant-ci/fireant/blob/docker/main.py#L158